### DISCUSSION TITLE
Watchface Maker

### DISCUSSION THREAD

**Discussion by ivndbt** [zepp-health/zeppos-forum#32] [Q&A]
I think some info are missing from the official guide. I can't understand how to select year format. I searched in the Wiki but I cannot find anything about formatting the output. I'm  trying to achieve a "YY" year format = only the last 2 digits of the year (2022  -> 22). Somebody  know how to do  it?

**Comment by ivndbt** [comment]
How da I change the preview image of the watchface if I have done some editing on the layout?

**Reply by sunnyBob** [reply]
change "format" to achieve a "YY" year format 
<img width="1439" alt="image" src="https://user-images.githubusercontent.com/18245566/210477412-768330a0-cc29-4b88-94e8-4db0cfc4df0c.png">

**Reply by ivndbt** [reply]
I already tried to check "abbreviate or not" option, but when I upload the watchface to the smartwatch it keep display the full format (4 digits).

**Reply by sunnyBob** [reply]
What smartwatch did you use for the test? Is it a non zeppos device?

**Reply by ivndbt** [reply]
I'm using an Amazfit GTR 2e.

**Reply by sunnyBob** [reply]
Thank you, the problem has been fixed, you can refresh your browser and add the date again
